ci/docker/build/Dockerfile

```diff
@@ -2,24 +2,22 @@ FROM ubuntu:16.04
 
 ENV DEPOT_TOOLS_PATH $HOME/depot_tools
 ENV ENGINE_PATH $HOME/engine
-
-RUN apt-get update
-RUN apt-get install -y git wget curl unzip python lsb-release sudo apt-transport-https
-
-RUN git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git $DEPOT_TOOLS_PATH
 ENV PATH $PATH:$DEPOT_TOOLS_PATH
 
-RUN mkdir --parents $ENGINE_PATH
+RUN apt-get update && \
+    apt-get install -y git wget curl unzip python lsb-release sudo apt-transport-https && \
+    git clone https://chromium.googlesource.com/chromium/tools/depot_tools.git $DEPOT_TOOLS_PATH && \
+    mkdir --parents $ENGINE_PATH
 WORKDIR $ENGINE_PATH
 ADD engine_gclient .gclient
```

Comment:
I think we can get rid of this and the gclient sync. We do this as part of the CI run now because it caused problems trying to use the baked in versions.